release-notes/1.0/Release-Notes-1.0.0.md

```diff
@@ -1,2 +1,103 @@
+| libuuid             | guid generation                                                  | Runtime       | libuuid1      | libuuid       | Part of OS            |
+| libcurl             | HTTP                                                             | Runtime       | libcurl3      | libcurl       | Part of OS            |
+| libssl & libcrypto  | cryptography                                                     | Runtime       | libssl1.0.0   | openssl-libs  | Openssl from homebrew |
+| libz                | deflate and gzip                                                 | Runtime       | zlib1g        |zlib           | Part of OS            |
+| liblttng            | runtime tracing                                                  | Diagnostic    | liblttng-ust0 | liblttng-ust0 | n/a                   |
+
+### Introduction of .NET Core API Documentation
+
+* Conceptual: http://docs.microsoft.com/dotnet
+* API: http://dotnet.github.io/api/
+
+### Production-quality .NET Core Runtime and Framework libraries
+
+* Background GC on Unix
+* Support for basic cross-process synchronization in the Mutex class on Unix
+* Reliability improvements
+* R2R Performance improvements for Generics
+* Portable PDB support in Exception.ToString() and System.Diagnostics.StackTrace
+* New SOS commands "!CLRStack -r"
+* Limited managed profiling support on Windows
+
+### BCL
+
+Below is a summary of API changes made since RC2. A complete and detailed listing of API changes can be seen in the [RC2 to Final API Diff](RC2-Final_API_diff.md).
+
+New .NET Core APIs
+
+* System.ComponentModel.TypeConverter - TypeDescriptor support
+* System.Data.Common - DBEnumerator
+* System.Diagnostics.Tracing - EventCounter
+* System.IO.FileSystem.Watcher - FileSystemWatcher.WaitForChanged
+* System.Net.Http - new properties on HttpClientHandler
+* System.Net.Sockets - add back Tcp|UdpClient.Client property
+* System.Reflection.Metadata - MetadataReaderProvider
+	* Many types have been removed from the 1.3 surface area and moved to 1.4 which remains in pre-release.  If you depend on these types update to the 1.4.0 pre-release package.
+* System.Resources.ReaderWriter - types are now in separate assemblies System.Resources.Reader and System.Resources.Writer
+* System.Runtime.CompilerServices.Unsafe - New static helper methods for dealing with unsafe pointers
+* System.Security.Cryptography
+    * ECDsa supports creating keys for arbitrary named and explicit curves (Windows 10, OS X, Linux)
+	* ECDsa supports importing and exporting keys via the platform-independent ECParameters type
+    * EnvelopedCms is now supported via the System.Security.Cryptography.Pkcs package
+* System.Security.SecureString
+* System.Text.RegularExpressions - precompiled regex
+
+### CLI
+
+* Offline support: now you don't have to be connected to the internet to write applications that target only the .NET Core runtime and libraries
```

Comment:
@akoeplinger , @blackdwarf 

How about - 

* Offline support: now you don't have to be connected to the internet to write applications that target only the .NET Core runtime and libraries. This means the core libraries are cached locally after running various dotnet commands the first time. Restoring packages which are not part of the Shared Framework do generally require and internet connection to gather the packages from NuGet.